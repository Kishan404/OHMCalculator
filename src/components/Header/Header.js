import React, { Component } from 'react';
import PropTypes from 'prop-types';
import logo from '../../resources/resister.svg';

const Header =(props) => {
    return (
        <div className="header">
            <h1 className="header-title">{props.hearder}</h1>
            <img src={logo} className="header-logo" alt="logo" />
            <p className="header-subtitle">{props.subTitle}</p>
        </div>
    );
}
Header.prototype = {
    hearder : PropTypes.string,
    subTitle : PropTypes.string,
}
export default Header;