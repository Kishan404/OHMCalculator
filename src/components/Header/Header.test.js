import React, { Component } from 'react';
const ReactTestRenderer = require('react-test-renderer');
import Header from './Header';

describe('Header result Snapshot test', () => {
    it('Should compare the component with a snapshot', () => {
      
    // const component = '<div class="header"><h1 class="header-title">4-Band Resistor ohm value  calculator</h1><img src="/static/media/resister.4013174d.svg" class="header-logo" alt="logo"><p class="header-subtitle">A usefull tool for reading resistor color code values</p></div>'
    // expect(component).toMatchSnapshot();
      
      const component = ReactTestRenderer.create(
        <Header
            hearder = {'4-Band Resistor ohm value calculator'}
            subTitle ={'A usefull tool for reading resistor color code values'} 
        />          
      );
      const json = component.toJSON();
      expect(json).toMatchSnapshot();
    })
  });
