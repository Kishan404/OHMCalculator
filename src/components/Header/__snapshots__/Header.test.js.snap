// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header result Snapshot test Should compare the component with a snapshot 1`] = `
<div
  className="header"
>
  <h1
    className="header-title"
  >
    4-Band Resistor ohm value calculator
  </h1>
  <img
    alt="logo"
    className="header-logo"
    src="resister.svg"
  />
  <p
    className="header-subtitle"
  >
    A usefull tool for reading resistor color code values
  </p>
</div>
`;
